generators/server/templates/src/main/java/package/config/DatabaseConfiguration.java.ejs

```diff
@@ -151,8 +156,11 @@ import java.util.UUID;
     <%_ } _%>
 @EnableTransactionManagement
 <%_ } _%>
+<%_ if (databaseType === 'neo4j') { _%>
+@Enable<% if (reactive) { %>Reactive<% } %>Neo4jRepositories(basePackageClasses = {RepositoryMarker.class})    
+<%_ } _%>   
 <%_ if (searchEngine === 'elasticsearch') { _%>
-@Enable<% if (reactive) { %>Reactive<% } %>ElasticsearchRepositories("<%= packageName %>.repository.search")
+@Enable<% if (reactive) { %>Reactive<% } %>ElasticsearchRepositories(basePackageClasses = {SearchRepositoryMarker.class})
```

Comment:
I wonder if the same workaround that is used for mongodb and couchbase would not work. Has it been tried?

Reply:
You mean with the filters? I didn't try but maybe it works too, worth a try.

Reply:
Yes, with the filters, sorry for not having been explicit...

Reply:
:heavy_check_mark: Done